pilot/platform/kube/inject/inject_test.go

```diff
@@ -1,566 +0,0 @@
-// Copyright 2017 Istio Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package inject
-
-import (
-	"bytes"
-	"os"
-	"reflect"
-	"testing"
-
-	"github.com/ghodss/yaml"
-	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	meshconfig "istio.io/api/mesh/v1alpha1"
-	"istio.io/istio/pilot/model"
-	"istio.io/istio/pilot/test/util"
-	"istio.io/istio/pkg/version"
-)
-
-func TestImageName(t *testing.T) {
-	want := "docker.io/istio/proxy_init:latest"
-	if got := InitImageName("docker.io/istio", "latest", true); got != want {
-		t.Errorf("InitImageName() failed: got %q want %q", got, want)
-	}
-	want = "docker.io/istio/proxy_debug:latest"
-	if got := ProxyImageName("docker.io/istio", "latest", true); got != want {
-		t.Errorf("ProxyImageName() failed: got %q want %q", got, want)
-	}
-	want = "docker.io/istio/proxy:latest"
-	if got := ProxyImageName("docker.io/istio", "latest", false); got != want {
-		t.Errorf("ProxyImageName(debug:false) failed: got %q want %q", got, want)
-	}
-}
-
-// Tag name should be kept in sync with value in platform/kube/inject/refresh.sh
-const unitTestTag = "unittest"
-
-// This is the hub to expect in platform/kube/inject/testdata/frontend.yaml.injected
-// and the other .injected "want" YAMLs
-const unitTestHub = "docker.io/istio"
-
-func TestIntoResourceFile(t *testing.T) {
```

Comment:
why not leave or git mv this test / would help making the change less scary ?

Reply:
`file{_test}.go` is for client-side file based injection. `webhook{_test.go` is for the server-side webhook based injection. `inject.go` is for the common code. `TestIntoResourceFile` is part of the client-side file based injection logic.

`git mv` operates at the file level AFAICT. I guess I could have `git mv inject_test.go file_test.go` and then made further edits from there, but I'm not sure the result would be that much cleaner.